src/js/controller/settings/exportimage/ZipExportController.js

```diff
@@ -71,4 +73,59 @@
   ns.ZipExportController.prototype.getPiskelName_ = function () {
     return this.piskelController.getPiskel().getDescriptor().name;
   };
+
+  ns.ZipExportController.prototype.createZipExportDesc = function (i18n) {
+    var templateData = {
```

Comment:
minor nitpick but I think you called it `templateValues` in other parts of the code right?